HLTrigger/Muon/plugins/HLTMuonTrkL1TFilter.h

```diff
@@ -21,6 +23,7 @@ class HLTMuonTrkL1TFilter : public HLTFilter {
                  trigger::TriggerFilterObjectWithRefs& filterproduct) const override;
 
 private:
+  mutable PropagateToMuonInterface propInt_;
```

Comment:
```suggestion
  const PropagateToMuonInterface propInt_;
```
I didn't see the need to use `mutable` anymore (here and in other similar places). No?

Reply:
True, that was an oversight, fixed.